drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -2250,7 +2250,8 @@ void RasterizerCanvasGLES3::_bind_canvas_texture(RID p_texture, RS::CanvasItemTe
 	RS::CanvasItemTextureRepeat repeat = ct->texture_repeat != RS::CANVAS_ITEM_TEXTURE_REPEAT_DEFAULT ? ct->texture_repeat : p_base_repeat;
 	ERR_FAIL_COND(repeat == RS::CANVAS_ITEM_TEXTURE_REPEAT_DEFAULT);
 
-	GLES3::Texture *texture = texture_storage->get_texture(ct->diffuse);
+	GLES3::Texture *main_texture = texture_storage->get_texture(ct->diffuse);
+	GLES3::Texture *light_texture = texture_storage->get_light_texture();
```

Comment:
Where did you get this method from?

Reply:
It is declared in `scene/3d/lightmap_gi.cpp` file, `LightmapGIData::get_light_texture()`

Reply:
I'm still trying to figure out exactly what to do here.

Reply:
But also why would you be looking at the 3D lightmap stuff when you're in 2D?

Reply:
You've made a great point, it doesn't make sense look into this method. I'll check it out later during my free time to come up with a reasonable implementation that make senses.